lib/src/core/repositories/groups_repository.dart

```diff
@@ -1,28 +1,13 @@
-import 'package:kana_to_kanji/src/core/services/database_service.dart';
+import 'package:kana_to_kanji/src/core/services/groups_service.dart';
 import 'package:kana_to_kanji/src/locator.dart';
 import 'package:kana_to_kanji/src/core/constants/alphabets.dart';
-import 'package:kana_to_kanji/src/core/constants/group_queries.dart';
 import 'package:kana_to_kanji/src/core/models/group.dart';
 
 class GroupsRepository {
-  final DatabaseService _databaseService = locator<DatabaseService>();
-
-  final List<Group> _groups = [];
```

Comment:
Why remove this section?

We should still have this section of the code. We don't want to pull from the DB every time we open a page that needs the groups. When this is needed the optional `reload` is there

Reply:
Ca n'est pas le but d'avoir la BD embarquée / dans le cache du navigateur de ne pas avoir plusieurs couches de cache?

Reply:
C'est toujours plus long d'aller chercher dans la DB.
En soit on garde ca en "cache" (ram) pour réduire le temps de chargement. La DB est là pour 2 raisons de mon point de vue:
1. hors ligne (future)
2. réduire la quantité d'appels à l'API qui peut-être long.

On prévoirait pas de mode hors-ligne, on pourrait completement retirer la DB en soit.